microsite/data/plugins/changelog.yaml

```
@@ -0,0 +1,12 @@
+---
+title: Changelog viewer
+author: RSC Labs
+authorUrl: https://rsoftcon.com/
+category: Discovery
+description: View changelogs for your components in Backstage.
```

I would mention that the default format for these is "Keep the changelog", maybe something like this:

```suggestion
description: View changelogs in "Keep the changelog" format for your components in Backstage.
```

Just mentioning this as Backstage itself uses changeset and that seems to be used by many adopters.

---

That's true, however it is possible to create own parser and pass to the plugin. I am thinking whether it is worth to mention it somehow to not give impression that "Keep the changelog" format is the only option (at this moment it is the only one with built-in support).
Maybe something like "View changelogs for your components in Backstage. Built-in support of "Keep the changelog" format" ?

---

That works, I do feel it's important to mention as I can easily see Adopters having the impression it would work with changeset out of the box as that is something they interact with regularly and may have also adopted 👍

---

I totally agree, thanks, I will correct soon.